GNUmakefile

```diff
@@ -52,18 +52,20 @@ test-compile:
 	fi
 	go test -c $(TEST) $(TESTARGS)
 
-dependencies:
-	@echo "Download go.mod dependencies"
-	@go mod download
-
-install-tools: dependencies
+BIN=$(CURDIR)/bin
+$(BIN)/%:
```

Comment:
This was related to the same change as explained in the comment on `.github/workflows/pr.yml`. As part of moving the installation of `tfplugindocs` to be inside the `terraform-provider-fastly` directory, I used the non-PHONY target to avoid needing to re-run it every time, if the binary already exists. By extension, the dynamic target seemed useful in case other dependencies were added, given that the grep/awk command was already capable of installing the whole `tools.go` file.

That's the rationale anyway - I'm happy to change it to use a PHONY `install-tools` target if you'd prefer, and if I'm not mistaken that would rebuild the tool on each invocation, which could be helpful if it were upgraded? That's assuming that you agree installing to a project-local `terraform-provider-fastly/bin` directory is a good idea anyway, which you might not - also ok!

Reply:
OK that makes sense. I think avoiding reinstalling the tool every time is better any way and we can suffer not being on the bleeding edge all the time.